testing/api/test_image.py

```diff
@@ -670,16 +670,18 @@ def test_sequences(fake_3dimage_sct):
 def test_add_suffix():
     assert msct_image.add_suffix('t2.nii', '_mean') == 't2_mean.nii'
     assert msct_image.add_suffix('t2.nii.gz', 'a') == 't2a.nii.gz'
-    assert msct_image.add_suffix('var/lib/usr/t2.nii.gz', 'sfx') == 'var/lib/usr/t2sfx.nii.gz'
-    assert msct_image.add_suffix('var/lib.version.3/usr/t2.nii.gz', 'sfx') == 'var/lib.version.3/usr/t2sfx.nii.gz'
+    assert msct_image.add_suffix(os.path.join('var', 'lib', 'usr', 't2.nii.gz'), 'sfx') \
+           == os.path.join('var', 'lib', 'usr', 't2sfx.nii.gz')
+    assert msct_image.add_suffix(os.path.join('var', 'lib.version.3', 'usr', 't2.nii.gz'), 'sfx') \
+           == os.path.join('var', 'lib.version.3', 'usr', 't2sfx.nii.gz')
 
 
 def test_splitext():
     assert msct_image.splitext('image.nii') == ('image', '.nii')
     assert msct_image.splitext('image.nii.gz') == ('image', '.nii.gz')
-    assert msct_image.splitext('folder/image.nii.gz') == ('folder/image', '.nii.gz')
+    assert msct_image.splitext('folder/image.nii.gz') == (os.path.join('folder', 'image'), '.nii.gz')
```

Comment:
I'm not sure if this is an intentional omission, given your other changes in this file.
```suggestion
    assert msct_image.splitext(os.path.join('folder', 'image.nii.gz')) == (os.path.join('folder', 'image'), '.nii.gz')
```